### ISSUE TITLE
State change is not read by screen reader 👀

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#264]
When the toggle button is clicked it's not announced that a state has changed—namely the 'Added to whitelist' action.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592354-5bfc307c-9643-11e8-97a5-f00f8638f8e2.png">

**Comment by gitgoap** [comment]
Is this issue open? @jagtalon
